src/main/java/org/traccar/reports/RouteReportProvider.java

```diff
@@ -66,7 +69,13 @@ public Collection<Position> getObjects(long userId, Collection<Long> deviceIds,
         }
         return result;
     }
+    private final Map<String, Integer> uniqueSheetsTitle = new HashMap<>();
```

Comment:
Please move to the top next to other variables. And also name doesn't really make sense. It just contains the number of times a name is included. Maybe just call `namesCount` or something like that.